### ISSUE TITLE
Pretrain emotion disc

### ISSUE THREAD

**Issue by sahilg06** [eeskimez/emotalkingface#7]
The classification-loss while pre-training the emotion-disc should go down to what value?

**Comment by eeskimez** [comment]
Hi, 

I don't remember the exact value, but the main idea is that the discriminator should learn to distinguish the emotions before the actual training. You can train it for a small number of epochs (5-7) as an initilization, which should be enough. Just make sure that the loss value is decreasing over time, otherwise, there might be a problem with the data.

Hope this helps!

Best,
Emre

**Comment by sahilg06** [comment]
Thanks, @eeskimez!

**Comment by sahilg06** [comment]
@eeskimez, are the weights of emotion-disc updated during actual training? or they are fixed after pre-training?
As if we will update its parameters during actual training, it might focus on visual artifacts instead of emotion.
